features/facebookImport/test/ministory/report-metadata.test.js

```diff
@@ -7,13 +7,12 @@ import ReportMetadataReport from "../../src/views/ministories/reportMetadata";
 import { createInteractedWithAdvertisersDataset } from "../datasets/interacted-with-advertisers-data";
 import { createLanguageSettingsData } from "../datasets/language-and-locale-data";
 import { createOffFacebookEventsSimpleData } from "../datasets/off-facebook-events-data";
-import { MINIMUM_FILE_SIZE } from "@polypoly-eu/poly-import";
+import { MINIMUM_FILE_SIZE, ZipFileMock } from "@polypoly-eu/poly-import";
 import { runAnalysisForExport } from "../utils/analyses-execution";
 import {
     expectActiveAnalysis,
     expectAnalysisSuccessStatus,
 } from "../utils/analysis-assertions";
-import { createMockedZip } from "../utils/data-creation";
```

Comment:
Can createMockedZip be removed now?

Reply:
You mean, from all over? It's used in some other places. I'd have to check.

Reply:
Right, it can be removed. It's essentially doing what the new ctor is doing. Only... It's all over, as said. I'll probably just refactor it to avoid extensive changes.